developers/integrations/llm-frameworks/haystack/index.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Haystack
+sidebar_position: 3
+---
+
+[Haystack](https://haystack.deepset.ai/) is a language model framework for build large language model applications 
```

Comment:
Typo and consider

[Haystack](https://haystack.deepset.ai/) is a framework for building large language model applications